lib/patreon/schemas/campaign.rb

```diff
@@ -0,0 +1,85 @@
+require_relative '../../enum/enum'
```

Comment:
the idea was that those are internal utils, not meant for consumption by clients of this library.. would you still recommend they get put on the public `.rb`?